html/elements/a.json

```
@@ -540,7 +540,7 @@
                 "version_added": "11.1"
               },
               "safari_ios": {
-                "version_added": false
+                "version_added": "13.0"
```

Omit the `.0` for major versions

```suggestion
                "version_added": "13"
```

---

Thanks, it was the wrong line as well, not sure how I messed that up.  
I have now properly changed the download attribute line.